src/co2.js

```diff
@@ -68,21 +68,32 @@ import { parseOptions } from "./helpers/index.js";
 
 class CO2 {
   constructor(options) {
+    let allowRatings = true;
```

Comment:
We're largely setting this `allowRatings `value based on the model being passed in the options object.

This suggests to me that we might want to set it on the model itself, and then set allowRatings as a boolean value like this.

```js
const allowRatings = !!this.model.allowRatings
```

If we do this this way, then we can rely on the value not being a a trusty `const` that we can largely trust not to change under us, and we'd only need to define it once.

The downside is that if some set the value on a model to `false` we might end up with a false positive if we just used the naive boolean check above.

I'd prefer to leave the decision to you, as it's not like we have loads of models to support, but it seemed worth raising.

Reply:
Cheers Chris. I've added a check to make sure the value passed in by the user is a boolean. For now, since we set the models there's no need to check if the type of value set inside the model itself. Probably something we'd want to do later though as part of #206